.rubocop.yml

```diff
@@ -121,6 +121,14 @@ Metrics/AbcSize:
 Metrics/BlockLength:
   Enabled: false
 
+Metrics/ClassLength:
```

Comment:
These cops were forcing people to smash arrays, hashes, and other things into as few lines as possible, which just impaired readability.